.github/workflows/ci.yml

```diff
@@ -305,7 +305,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [14, 18]
-        mssql-version: [2017, 2019]
+        mssql-version: [2017, 2019, 2022]
```

Comment:
```suggestion
        mssql-version: [2017, 2022]
```

I think we should consider not including `2019` in our regular CI as it is already very long, until we optimize our tests to make them run faster and/or have 'intermediate' versions be run as smoke tests once a week?

Reply:
Yeah, I kept this in because 2022 is still in public preview but let's remove it now before we forget about it

Reply:
For running the 'intermediate' versions as smoke tests, I have plans on that which I still need to finish working out. But mainly the idea is indeed to run quite some additional configurations once a week

Reply:
Oh right I forgot you already told me